util/types/bit.go

```diff
@@ -104,3 +105,33 @@ func ParseBit(s string, width int) (Bit, error) {
 
 	return Bit{Value: n, Width: width}, nil
 }
+
+// ParseBitValue parses the binary string for bit type into uint64.
+func ParseBitValue(s string, width int) (uint64, error) {
```

Comment:
We already have a func named ParseBit to parse binary string to uint64.
It seems the implementation of this func is used to parse a hex string to uint64?
If so, the width in line 117 may be width = len(b) * 8?